testing/trino-product-tests/src/main/java/io/trino/tests/cli/TestTrinoLdapCli.java

```diff
@@ -260,6 +265,17 @@ public void shouldFailForUserWithColon()
         skipAfterTestWithContext();
     }
 
+    @Test(groups = {LDAP_AND_FILE_CLI, PROFILE_SPECIFIC_TESTS}, timeOut = TIMEOUT)
+    public void shouldRunQueryWithFile()
+            throws IOException
+    {
+        ldapUserPassword = fileUserPassword;
+        launchTrinoCliWithServerArgument();
+        trino.waitForPrompt();
+        trino.getProcessInput().println("select * from hive.default.nation;");
+        assertThat(trimLines(trino.readLinesUntilPrompt())).containsAll(nationTableInteractiveLines);
```

Comment:
Please use JDBC for testing. Please make sure you test two authenticators. Please also add some negative tests where user was not able to authenticate.

Reply:
Please next time make 3 remarks instead of one.
2.This test is for the file authenticator, `shouldRunQueryWithLdap` is for LDAP. I don't see reason to repeat any of them.
3.Every failing test in this class doe this, e.g. `shouldFailQueryForWrongLdapPassword`

Reply:
Using `TrinoLdapCLI` for your use case is not a best fit. These tests are targeted for something else. I think need a dedicated set of tests.

Reply:
I use both CLI and JDBC now.